composer.json

```diff
@@ -4,8 +4,10 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
+        "drupal/crop": "^2.1",
```

Comment:
Disappointing that `focal_point` doesn't seem to be aware of composer and you have to add this manually.